code/__DEFINES/clothing.dm

```diff
@@ -37,3 +37,32 @@
 /// How much integrity does a shirt lose every time we bite it?
 #define MOTH_EATING_CLOTHING_DAMAGE 15
 */
+//Whitelist for the suit storage slot on medical suits
+#define MEDICAL_SUIT_ALLOWED_ITEMS list( \
+    /obj/item/scalpel, \
+    /obj/item/cautery, \
+    /obj/item/hemostat, \
+    /obj/item/retractor, \
+    /obj/item/surgicaldrill, \
+    /obj/item/circular_saw, \
+    /obj/item/analyzer, \
+    /obj/item/sensor_device, \
+    /obj/item/stack/medical, \
+    /obj/item/dnainjector, \
+    /obj/item/reagent_containers/dropper, \
+    /obj/item/reagent_containers/syringe, \
+    /obj/item/reagent_containers/hypospray, \
+    /obj/item/healthanalyzer, \
+    /obj/item/flashlight/pen, \
+    /obj/item/reagent_containers/glass/bottle, \
+    /obj/item/reagent_containers/glass/beaker, \
+    /obj/item/reagent_containers/pill, \
+    /obj/item/storage/pill_bottle, \
+    /obj/item/paper, \
+    /obj/item/melee/classic_baton/telescopic, \
+    /obj/item/toy, \
+    /obj/item/storage/fancy/cigarettes, \
+    /obj/item/lighter, \
+    /obj/item/tank/internals/emergency_oxygen, \
+    /obj/item/tank/internals/plasmaman \
+)
```

Comment:
Entries in the list look like they should be tabbed once with tabs, these are currently tabbed with spaces